plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/struct/DBSDataBulkLoader.java

```diff
@@ -32,7 +33,7 @@
     interface BulkLoadManager extends AutoCloseable {
         void addRow(@NotNull DBCSession session, @NotNull Object[] attributeValues) throws DBCException;
 
-        void flushRows(@NotNull DBCSession session) throws DBCException;
+        DBCStatistics flushRows(@NotNull DBCSession session) throws DBCException;
```

Comment:
Missing `@Nullable` annotation on the return type.

Reply:
This change was reverted in a new commit.